cc_plugin_glider/util.py

```diff
@@ -57,7 +57,7 @@ def _check_dtype(dataset, var_name):
     return (score, out_of, messages)
 
 
-def _check_variable_attrs(dataset, var_name, required_attributes=None):
+def _check_variable_attrs(dataset, var_name, required_attributes=None, options=None):
```

Comment:
Let's document this new kw in the docstring below. In my mind I was think that this added options for the function but it is adding variables that the checks are optional, right? Or maybe variables that you want to skip? If the latter, maybe let's rename this kw to "skip_variables" and document that the user should pass a list of variables to skip. That way we can drop the complex "no_var-name" check below and just drop whatever is in the list.

Does that make sense?